### ISSUE TITLE
Signaling Workflows in elsa3

### ISSUE THREAD

**Issue by zesmaili** [elsa-workflows/elsa-core#5029]
hi 
In version 2 there were activities named "Send Signal" and "Received Signal".
But in version 3 these activities are missing, how can I access them?

**Comment by zesmaili** [comment]
> In Elsa version 3, the activities that were known as "Send Signal" and "Signal Received" in version 2 have been replaced with different concepts and activities. The signaling mechanism in Elsa 3 is not the same as in Elsa 2, and you might need to use different activities to achieve similar functionality.
> 
> For instance, in Elsa 3, you might use the "Publish Event" and "Event" activities for similar purposes. The "Publish Event" activity allows you to publish an event with a name and correlation ID, while the "Event" activity acts as an event handler that can continue the workflow if an event is raised.
> 
> If you are looking to implement custom logic before suspending the workflow, you may need to create custom activities. The user from the Stack Overflow question titled "Custom 'signal received' activity not working" was attempting to create a custom activity to mimic the "Signal Received" behavior with additional logic.
> 
> For more information on the available activities in Elsa 3 and how to use them, you can refer to the official Elsa documentation, specifically the sections on primitive activities, flow activities, and branching activities. If you need to create custom activities, you can follow the guidelines provided in the documentation and examples from the community.
> 
> If you are migrating from Elsa 2 to Elsa 3 and need specific guidance on how to replace the "Send Signal" and "Signal Received" activities, it would be best to consult the Elsa 3 documentation or reach out to the Elsa community for advice on how to achieve similar functionality with the new version.

One of the applications of these activities in my work is to go to the next state by a human action, which is possible in version 2 by the "State" and "signal" activities.
But now I don't know how to do it
please guide me

**Comment by AuGuima** [comment]
> One of the applications of these activities in my work is to go to the next state by a human action, which is possible in version 2 by the "State" and "signal" activities. But now I don't know how to do it please guide me

Take a look at this Elsa code, I think it might help you: https://github.com/elsa-workflows/elsa-core/tree/main/samples/aspnet/Elsa.Samples.AspNet.DocumentApproval

**Comment by zesmaili** [comment]
> > One of the applications of these activities in my work is to go to the next state by a human action, which is possible in version 2 by the "State" and "signal" activities. But now I don't know how to do it please guide me
> 
> Take a look at this Elsa code, I think it might help you: https://github.com/elsa-workflows/elsa-core/tree/main/samples/aspnet/Elsa.Samples.AspNet.DocumentApproval

Yes, it is exactly the same example, but I want to be able to create such workflows using UI and Run-Time, but all the activities used in this example have been replaced with this similar activity, which has less functionality.
